### PR TITLE
change a file content of a recived shared

### PR DESCRIPTION
**Pull request by ScharfViktor**
Describe the bug
A test is written that demonstrates the error. The test will work when the bug is fixed

Steps to reproduce

User1 creates file and shares file to User2
User2 accepts file and try to upload file with some name
Or run test: yarn run test:acceptance:ocis tests/acceptance/features/webUIUserJourney/journey1.feature:29

Expected behavior
Created new version

Actual behavior
403 Forbinden 

Screen Shot 2021-07-15 at 8 39 30 PM

Setup
Please describe how you started the server and provide a list of relevant environment variables.

OCIS_VERSION=v1.9.0 reachable at https://localhost:9200
BRANCH=v1.9.0-rc1

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalGroupsSharingIndicator</strong> https://drone.owncloud.com/owncloud/web/17469/23/1
The following scenarios passed on retry:
- webUISharingInternalGroupsToRootSharingIndicator/shareWithGroups.feature:104

**Comment by ownclouders** [comment]
Results for <strong>oCISTrashbinUploadMoveJourney</strong> https://drone.owncloud.com/owncloud/web/17469/62/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

File: tests/acceptance/features/webUIUserJourney/journey1.feature [hunk 1]
```diff
@@ -25,3 +25,12 @@ Feature: User Journey 1
     And the user restores file "new-lorem2.txt" from the trashbin using the webUI
     And the user browses to the files page
     And file "new-lorem2.txt" should be listed on the webUI
+
+  Scenario: change a file content of a recived shared
```
Comment (phil-davis) [inline tests/acceptance/features/webUIUserJourney/journey1.feature:29]:
```suggestion
  Scenario: change the file content of a received shared file
```

File: tests/acceptance/features/webUIUserJourney/journey1.feature [hunk 2]
```diff
@@ -25,3 +25,12 @@ Feature: User Journey 1
...
+    Given user "Brian" has created file "lorem.txt"
+    And user "Brian" has shared file "lorem.txt" with user "Alice" with "all" permissions
+    And user "Alice" has accepted the share "lorem.txt" offered by user "Brian"
+    When user "Alice" has logged in using the webUI
+    And the user opens folder "Shares" using the webUI
+    When the user uploads overwriting file "lorem.txt" using the webUI
```
Comment (phil-davis) [inline tests/acceptance/features/webUIUserJourney/journey1.feature:35]:
```suggestion
    And user "Alice" has logged in using the webUI
    When the user opens folder "Shares" using the webUI
    And the user uploads overwriting file "lorem.txt" using the webUI
```

I suggest that you put a single `When` where the "action" starts to really happen.
The steps with "has done something" are intend to be used in the `Given`...`And` section. The grammar sound better in a sentence starting with "Given".

**Review by phil-davis — COMMENTED** [review]
I am surprised that we do not already have a test scenario like this. Maybe we don't?

File: tests/acceptance/features/webUIUserJourney/journey1.feature (hunk 2 above)
Comment (phil-davis) [reply tests/acceptance/features/webUIUserJourney/journey1.feature:35]:
```suggestion
    And user "Alice" has logged in using the webUI
    And the user has opened folder "Shares" using the webUI
    When the user uploads overwriting file "lorem.txt" using the webUI
```

This is also a way that the "And" and "When" could be organized. The real "action" step is just the "uploads overwriting".

**Comment by ownclouders** [comment]
Results for <strong>oCISTrashbinUploadMoveJourney</strong> https://drone.owncloud.com/owncloud/web/17485/62/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Review by dpakach — COMMENTED** [review]
Looks good. Since this is a test that is expected to fail, this also needs to be added to expected failure file so that the build passes.

**Comment by ownclouders** [comment]
Results for <strong>oC10Files1</strong> https://drone.owncloud.com/owncloud/web/17513/11/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oC10Files1</strong> https://drone.owncloud.com/owncloud/web/17518/11/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oC10Files1</strong> https://drone.owncloud.com/owncloud/web/17523/11/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ScharfViktor** [comment]
opened a new https://github.com/owncloud/web/pull/5548

**Pull request closed without merging**